packages/renderer/src/lib/image/ImageActions.svelte

```diff
@@ -78,6 +88,14 @@ if (dropdownMenu) {
       icon="{faLayerGroup}" />
   {/if}
 
+  {#each contributions as menu}
+    <ListItemButtonIcon
+      title="{menu.title}"
+      onClick="{() => executeContribution(menu)}"
+      menu="{dropdownMenu}"
+      icon="{faArrowUp}" />
```

Comment:
FWIU, all commands contributed will receive a ⬆️ icon which is not what we would expect

Reply:
Yes but my understanding is that icon support for contribution should be in a separate PR so I choose this one as it's required

Reply:
I would prefer to not have icon rather than applying an upload icon to all contributed actions. Or to have a more generic icon maybe

Reply:
https://fontawesome.com/icons/ellipsis-vertical ?